docusaurus/docs/Android/04-compose/04-message-components/05-attachments-picker.mdx

```diff
@@ -1,10 +1,10 @@
 # AttachmentsPicker
 
-The `AttachmentsPicker` component allows users to pick media, file and media capture attachments to send to the chat. The picker is a **bound component**, that loads all the data and prepares it for the user.
+The `AttachmentsPicker` component allows users to pick media, file or media capture attachments to send to the chat. The picker is a **bound component** that loads all the data and prepares it for the user.
 
 Internally, it sets up the following components:
 
-* Picker options: The header in the picker that shows different types of attachment options people can choose from — media, files and media capture, as well as the **Send** button.
+* Picker options: The header in the picker that shows different types of attachment options people can choose from — media, files or media capture, as well as the **Confirm selection** button.
```

Comment:
Put these in bold maybe: "picker options", "Image picker", "File picker", "Media capture"